tapchannel/aux_funding_controller.go

```diff
@@ -1058,6 +1058,18 @@ func (f *FundingController) completeChannelFunding(ctx context.Context,
 
 	log.Debugf("Funding PSBT pre funding: %s", spew.Sdump(fundingPsbt))
 
+	// Before we proceed, we'll make sure the fee rate we're using is above
+	// the min relay fee.
+	minRelayFee, err := f.cfg.ChainWallet.MinRelayFee(ctx)
+	if err != nil {
+		return nil, fmt.Errorf("unable to establish min_relay_fee: %w",
+			err)
+	}
+	if minRelayFee > fundingState.feeRate.FeePerKWeight() {
```

Comment:
micro-nit: I would expect the condition to read "if my transaction's fee is _smaller_ than the minimum relay fee, I have a problem". So perhaps we can rewrite it that way?